### PR TITLE
prep repo for testing in jestjs

### PR DESCRIPTION
**Pull request by cpprian**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "coverage": "jest --coverage --watchAll",
```
Comment (kasia251011) [inline package.json:33]: are you sure that it works?

File: package.json [hunk 2]
```diff
@@ -64,5 +65,20 @@
...
+      "!src/**/*.{stories,story,spec,test}.{js,jsx,ts,tsx}"
+    ],
+    "coverageThreshold": {
+      "global": {
+        "branches": 80,
+        "functions": 80,
+        "lines": 80,
+        "statements": -10
+      }
+    }
```
Comment (kasia251011) [inline package.json:82]: As I know, app created with create-react-app dont need special configuration for jest:
https://create-react-app.dev/docs/running-tests/#coverage-reporting

If you wanted to override deafult configuration, please describe what you wanted to achieve.

**Review by kasia251011 — CHANGES_REQUESTED** [review]
.

File: package.json (hunk 1 above)
Comment (cpprian) [reply package.json:33]: yes, but we can stay with react-scripts cli

File: package.json (hunk 2 above)
Comment (cpprian) [reply package.json:82]: collectCoverageFrom -> We want to test only feature components, not configurations or types
coverageThreshold -> It enables you to define minimum acceptable coverage levels for different types of code
preset -> Helps you define js syntax, for react-scripts doesn't work

**Pull request closed without merging**